src/style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+html {
+  --main-color: #060b35;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 404px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product-photo {
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+}
+
+.product-description {
+  display: flex;
+  flex-direction: column;
+  margin-top: 40px;
+  width: 166px;
+}
```

Comment:
use only margin-bottom to all card's elements. When you use only margin-bottom you avoid margin colapse